easybuild/easyconfigs/s/SWASH/SWASH-3.14-intel-2015b-mpi.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'MakeCp'
+
+name = 'SWASH'
+version = '3.14'
+versionsuffix = '-mpi'
```

Comment:
I have all three variants here (including serial), so I want to make it very clear which is which.